src/Data/List/Relation/Binary/Permutation/Homogeneous.agda

```diff
@@ -59,3 +59,12 @@ map R⇒S (refl xs∼ys)         = refl (Pointwise.map R⇒S xs∼ys)
 map R⇒S (prep e xs∼ys)       = prep (R⇒S e) (map R⇒S xs∼ys)
 map R⇒S (swap e₁ e₂ xs∼ys)   = swap (R⇒S e₁) (R⇒S e₂) (map R⇒S xs∼ys)
 map R⇒S (trans xs∼ys ys∼zs)  = trans (map R⇒S xs∼ys) (map R⇒S ys∼zs)
+
+------------------------------------------------------------------------
+-- Steps moves here because it is representation-dependent
```

Comment:
This feels like a GitHub comment rather than a code comment. Maybe just a simple comment that says `Measures the number of constructors, can be useful for termination proofs` rather than a section heading?